python/lib/dependabot/python/file_parser/python_requirement_parser.rb

```diff
@@ -83,7 +83,12 @@ def pip_compile_python_requirement
         def python_version_file_version
           return unless python_version_file
 
-          file_version = python_version_file.content.strip
+          # read the content, split into lines and remove any lines with '#'
+          content_lines = python_version_file.content.each_line.map do |line|
```

Comment:
I don't think we need to call `.map` on each_line, it already takes a block that gives us a string, so this could be simplified to:

```suggestion
          content_lines = python_version_file.content.each_line do |line|
```